rising/transforms/functional/painting.py

```diff
@@ -0,0 +1,80 @@
+import torch
+
+__all__ = ["local_pixel_shuffle", "random_inpainting", "random_outpainting"]
+
+
+def local_pixel_shuffle(data: torch.Tensor, n: int = -1) -> torch.Tensor:
+
+    batch_size, channels, img_rows, img_cols, img_deps = data.size()
+
+    if n < 0:
+        n = int(1000*channels) # changes ~ 12.5% of voxels
+    for b in range(batch_size):
+        for _ in range(n):
+            c = torch.randint(0,channels-1, (1,))
+            block_size_x = torch.randint(2, img_rows // 10, (1,))
+            block_size_y = torch.randint(2, img_cols // 10, (1,))
+            block_size_z = torch.randint(2, img_deps // 10, (1,))
+            x = torch.randint(0, int(img_rows - block_size_x), (1,))
+            y = torch.randint(0, int(img_cols - block_size_y), (1,))
+            z = torch.randint(0, int(img_deps - block_size_z), (1,))
+
+            window = data[b, c, x:x + block_size_x,
+                            y:y + block_size_y,
+                            z:z + block_size_z,
+                        ]
+            idx = torch.randperm(window.numel())
+            window = window.view(-1)[idx].view(window.size())
+
+            data[b, c, x:x + block_size_x,
+                          y:y + block_size_y,
+                          z:z + block_size_z] = window
+
+    return data
```

Comment:
```suggestion
def local_pixel_shuffle(data: torch.Tensor, n: int = -1, block_size: tuple=(0,0,0), rel_block_size: float = 0.1) -> torch.Tensor:

    batch_size, channels, img_rows, img_cols, img_deps = data.size()

    if n < 0:
        n = int(1000*channels) # changes ~ 12.5% of voxels
    for b in range(batch_size):
        for _ in range(n):
            c = torch.randint(0,channels-1, (1,))
            
            (block_size_x, block_size_y, block_size_z)  = (torch.tensor([size]) for size in block_size)
            
            if rel_block_size > 0:
                block_size_x = torch.randint(2, int(img_rows * rel_block_size), (1,))
                block_size_y = torch.randint(2, int(img_cols * rel_block_size), (1,))
                block_size_z = torch.randint(2, int(img_deps * rel_block_size), (1,))

            x = torch.randint(0, int(img_rows - block_size_x), (1,))
            y = torch.randint(0, int(img_cols - block_size_y), (1,))
            z = torch.randint(0, int(img_deps - block_size_z), (1,))

            window = data[b, c, x:x + block_size_x,
                            y:y + block_size_y,
                            z:z + block_size_z,
                        ]
            idx = torch.randperm(window.numel())
            window = window.view(-1)[idx].view(window.size())

            data[b, c, x:x + block_size_x,
                          y:y + block_size_y,
                          z:z + block_size_z] = window

    return data
```

Suggestion: Give user the option to define patch sizes themselves, either as absolute sizes or relative to the image size. My Implementation overwrites absolute sizes with relative sizes, if the relative size is > 0 (default = 0.1 to keep default behavior).

Reply:
I think this is a good idea. However, for users, the settings will be difficult to understand -> it needs to be clearer to the user how this works. E.g., at the moment, if the user sets block_size, but the rel_block_size is left to default, block_size will be ignored.
E.g., at least some comments, and perhaps some improvements in the input handling would be great